source/includes/common-replication.rst

```diff
@@ -30,6 +30,8 @@ MinIO relies on the immutability protections provided by :ref:`versioning <minio
 Use :mc-cmd:`mc version info` to validate the versioning status of both the source and remote buckets. 
 Use the :mc-cmd:`mc version enable` command to enable versioning as necessary.
 
+If you exclude a prefix or folder from versioning within the source bucket, MinIO cannot replicate objects within that folder or prefix.
```

Comment:
I'm waffling on including this at all, as it calls attention to versioning exclusion in the first place.

Reply:
Unless we are likely to change "no versioning, no replication," my thought is better to be clear about it.

Reply:
Taking Daryl's suggestions with a tweak

```suggestion
MinIO cannot replicate objects in bucket prefixes that you excluded from versioning.
```